DevCycle.SDK.Server.Common/Model/Local/Variable.cs

```diff
@@ -104,29 +102,27 @@ public static Variable<T> InitializeFromVariable(Variable<T> variable, string ke
         /// <value>Unique key by Project, can be used in the SDK / API to reference by &#x27;key&#x27; rather than _id.</value>
         [DataMember(Name = "key")]
         public string Key { get; set; }
-        
-        [DataMember(Name = "isDefaulted")]
-        public bool IsDefaulted { get; set; }
+
+        [DataMember(Name = "isDefaulted")] public bool IsDefaulted { get; set; }
         public string EvalReason { get; set; }
 
         public static TypeEnum DetermineType(T variableValue)
         {
             TypeEnum typeEnum;
-    
+
             try
             {
                 var baseType = variableValue.GetType();
-                var type = baseType.GetExtendedType();
```

Comment:
Why wern't we using the `baseType` before?

Reply:
Because of ease of use - not sure - will check gblame

Reply:
yea this semes like the most important thing to understand here, as this could change the functionality of the SDK.

Reply:
The answer is that it was to prevent a verbose switch setup. No specific technical reason. Just code cleanliness.

Reply:
yeah, it looks like that GetExtendedType() came from the TypeSupport library that we needed to remove. It seems like it is just a wrapper that adds additional interrogation options for types. The only feature we were using is the IsNumericType() utility which it looks like Jamie just replaced with a new custom implementation.

Reply:
Further answer - it was part of the autogenerated sdks